### ISSUE TITLE
[I3_PORTS] CentOS7 freetype error (Trac #1700)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1692]
On CentOS7, I can't install Geant4 or Root from ports because of a freetype error:

```text
g++  -pipe -m64 -Wshadow -Wall -W -Woverloaded-virtual -fPIC -Iinclude    -pthread -Wno-deprecated-declarations -I. -I/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_7_x86_64/i3ports/var/db/dports/build/file._cvmfs_icecube.opensciencegrid.org_py2-v2_RHEL_7_x86_64_i3ports_var_db_dports_sources_rsync.code.icecube.wisc.edu_icecube-tools-ports_science_root_5.34.18/work/root/cint/cint/inc  -o misc/memstat/src/G__MemStat.o -c misc/memstat/src/G__MemStat.cxx
In file included from asfont.c:67:0:
../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:21:2: error: #error "`ft2build.h' hasn't been included yet!"
 #error "`ft2build.h' hasn't been included yet!"
  ^
../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:22:2: error: #error "Please always use macros to include FreeType header files."
 #error "Please always use macros to include FreeType header files."
  ^
../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:23:2: error: #error "Example:"
 #error "Example:"
  ^
../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:24:2: error: #error "  #include <ft2build.h>"
 #error "  #include <ft2build.h>"
  ^
../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:25:2: error: #error "  #include FT_FREETYPE_H"
 #error "  #include FT_FREETYPE_H"
  ^
make[1]: *** [asfont.o] Error 1
```

Note that the header in question is available at `/usr/include/ft2build.h`

Migrated from https://code.icecube.wisc.edu/ticket/1700
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "On CentOS7, I can't install Geant4 or Root from ports because of a freetype error:\n\n{{{\ng++  -pipe -m64 -Wshadow -Wall -W -Woverloaded-virtual -fPIC -Iinclude    -pthread -Wno-deprecated-declarations -I. -I/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_7_x86_64/i3ports/var/db/dports/build/file._cvmfs_icecube.opensciencegrid.org_py2-v2_RHEL_7_x86_64_i3ports_var_db_dports_sources_rsync.code.icecube.wisc.edu_icecube-tools-ports_science_root_5.34.18/work/root/cint/cint/inc  -o misc/memstat/src/G__MemStat.o -c misc/memstat/src/G__MemStat.cxx\nIn file included from asfont.c:67:0:\n../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:21:2: error: #error \"`ft2build.h' hasn't been included yet!\"\n #error \"`ft2build.h' hasn't been included yet!\"\n  ^\n../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:22:2: error: #error \"Please always use macros to include FreeType header files.\"\n #error \"Please always use macros to include FreeType header files.\"\n  ^\n../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:23:2: error: #error \"Example:\"\n #error \"Example:\"\n  ^\n../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:24:2: error: #error \"  #include <ft2build.h>\"\n #error \"  #include <ft2build.h>\"\n  ^\n../../../../graf2d/freetype/src/freetype-2.3.12/include/freetype/freetype.h:25:2: error: #error \"  #include FT_FREETYPE_H\"\n #error \"  #include FT_FREETYPE_H\"\n  ^\nmake[1]: *** [asfont.o] Error 1\n}}}\n\nNote that the header in question is available at `/usr/include/ft2build.h`", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "tools/ports", 
    "summary": "[I3_PORTS] CentOS7 freetype error", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-05-12T16:23:08", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160621T19:15:01: 
* david.schultz commented:

> This has been magically fixed, possibly by the same thing that fixed #1745.


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas changed milestone from "June 2016" to ""
* olivas commented:

> Milestone June 2016 deleted
